data/campaigns/Heir_To_The_Throne/units/Dark_Queen.cfg

```diff
@@ -60,9 +64,15 @@
             name=scepter
         [/filter_attack]
         start_time=-200
+        swoosh_start_time=-50
         [frame]
-            image="units/human-queen-staff-[1~4].png:[100*3,75]"
+            image="units/human-queen.png:100,units/human-queen-attack.png:200,units/human-queen.png:1"
         [/frame]
+        [swoosh_frame]
+            image="units/human-queen-swoosh.png:100,misc/blank-hex.png:1"
+	    auto_vflip=no
```

Comment:
Currently he uses the unit image. It's small and, frankly, a bit funky looking.

As a Sea Orc, he has a line when he changes (turn 4), and another when he's killed.

And, to be honest, I never noticed he actually changed appearance until I was working on the campaign. So I tried to get the camera on him when he changes from a normal Orc to a Sea Orc so there's at least a small chance the change on-screen will draw the player's eye.